SWLOR.Game.Server/Feature/GuiDefinition/ViewModel/HoloNetViewModel.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Webhook;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.GuiService;
+
+namespace SWLOR.Game.Server.Feature.GuiDefinition.ViewModel
+{
+    public class HoloNetViewModel : GuiViewModelBase<HoloNetViewModel, GuiPayloadBase>
+    {
+        public const int MaxHoloNetTextLength = 800;
+
+        protected override void Initialize(GuiPayloadBase initialPayload)
+        {
+            HoloNetText = string.Empty;
+            WatchOnClient(model => model.HoloNetText);
+        }
+        public string HoloNetText
+        {
+            get => Get<string>();
+            set
+            {
+                Set(value);
+            }
+        }
+
+        public Action OnClickSubmit() => () =>
+        {
+            if (string.IsNullOrWhiteSpace(HoloNetText))
+            {
+                return;
+            }
+
+            var message = HoloNetText;
+
+            if (message.Length > 600)
+            {
+                SendMessageToPC(Player, "Your HoloNet broadcast was too long. Please shorten it to no longer than 600 characters and resubmit the broadcast. For reference, your message was: \"" + message + "\"");
+                return;
+            }
+            var area = GetArea(Player);
+            var position = GetPosition(Player);
+
+            var url = Environment.GetEnvironmentVariable("SWLOR_HOLONET_WEBHOOK_URL");
+
+            if (string.IsNullOrWhiteSpace(url))
+            {
+                SendMessageToPC(Player, ColorToken.Red("ERROR: Unable to send the HoloNet broadcast because server admin has not specified the 'SWLOR_HOLONET_WEBHOOK_HOST' and 'SWLOR_HOLONET_WEBHOOK_PATH' environment variables."));
+                return;
+            }
+
+            var price = 100;
+
+            if (GetGold(Player) < price)
+            {
+                SendMessageToPC(Player, ColorToken.Red("Insufficient credits to make this HoloNet broadcast."));
+                return;
+            }
+
+            else AssignCommand(Player, () => TakeGoldFromCreature(price, Player, true));
```

Comment:
You are already exiting early with the return statement on line 58. The "else" can be deleted as it is unnecessary.